lib/Commands/ClearBuildCacheCommand.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Website\Commands;
+
+use Doctrine\Website\Projects\ProjectRepository;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Filesystem\Filesystem;
+use function sprintf;
+
+class ClearBuildCacheCommand extends Command
+{
+    /** @var ProjectRepository */
+    private $projectRepository;
+
+    /** @var Filesystem */
+    private $filesystem;
+
+    /** @var string */
+    private $rootDir;
+
+    /** @var string */
+    private $env;
+
+    public function __construct(ProjectRepository $projectRepository, Filesystem $filesystem, string $rootDir, string $env)
+    {
+        parent::__construct();
+
+        $this->projectRepository = $projectRepository;
+        $this->filesystem        = $filesystem;
+        $this->rootDir           = $rootDir;
+        $this->env               = $env;
+    }
+
+    protected function configure() : void
+    {
+        $this
+            ->setName('clear-build-cache')
+            ->setDescription('Clear the build cache.')
+            ->addArgument(
+                'build-dir',
+                InputArgument::OPTIONAL,
+                'The directory where the build repository is cloned.'
+            )
+        ;
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output) : void
+    {
+        $buildDir = $input->getArgument('build-dir');
+
+        if ($buildDir === null) {
+            $buildDir = sprintf('%s/build-%s', $this->rootDir, $this->env);
```

Comment:
You can put this directly as a default value to `->addArgument()` inside `configure()` (4th parameter IIRC).